### PR TITLE
Minjiaz/mos doc

### PR DESCRIPTION
**Pull request by awan-10**

### CODE REVIEW

File: docs/_tutorials/mixture-of-experts-nlg.md [hunk 1]
```diff
@@ -51,3 +51,18 @@ Regarding training data, we are not able to release our internal data but any pu
 | 350M+MoE-128, internal data | 0.6365 | 0.7399 | 0.5988 | 0.3569 | 0.1630 | 0.0473 |
 
 Table 1: Zero-shot evaluation results (last six columns) for different dense and MoE NLG models. All zero-shot evaluation results use the accuracy metric.
+
+### 2.4. Reducing the model size with MoS
```
Comment (awan-10) [inline docs/_tutorials/mixture-of-experts-nlg.md:55]: Make title more clear on training

File: docs/_tutorials/mixture-of-experts-nlg.md [hunk 2]
```diff
@@ -51,3 +51,18 @@ Regarding training data, we are not able to release our internal data but any pu
...
+MoS, standing for Mixture-of-Students, is a staged distillation-based technique for compressing large MoE models. MoS further reduces the model size by 12.5%, leading to up 3.7x model size reduction when combined with PR-MoE over the standard MoE. The reduced model size helps reduce the latecy and cost during inference. To train an MoS model, one needs to specify a few additional parameters. We will use PR-MoE as an example:
+
+`--mos`: This would enable Mixture-of-Students via knowledge distillation. 
+
+`--load-teacher`: This specifies the path to the teacher model checkpoint.
```
Comment (awan-10) [inline docs/_tutorials/mixture-of-experts-nlg.md:60]: mandatory argument

File: docs/_tutorials/mixture-of-experts-nlg.md [hunk 3]
```diff
@@ -51,3 +51,18 @@ Regarding training data, we are not able to release our internal data but any pu
 | 350M+MoE-128, internal data | 0.6365 | 0.7399 | 0.5988 | 0.3569 | 0.1630 | 0.0473 |
 
 Table 1: Zero-shot evaluation results (last six columns) for different dense and MoE NLG models. All zero-shot evaluation results use the accuracy metric.
+
```
Comment (awan-10) [inline docs/_tutorials/mixture-of-experts-nlg.md:54]: add mos row to table

**Comment by awan-10** [comment]
This looks good to me. After format fixes, we can merge it. @minjiaz - please do so.

**Review by minjiaz — APPROVED** [review]
Reviewed the doc together with Ammar.

**Pull request merged**